hack/build-images.sh

```diff
@@ -125,7 +122,7 @@ function image {
 ( image openshift/origin-docker-registry       images/dockerregistry ) &
 ( image openshift/origin-egress-router         images/router/egress ) &
```

Comment:
I didn't realize you had `image` AND `image-build` ... `image` is not a super clear method name, from name alone I have no idea what differentiates it from `image-build`.